README.md

```
@@ -692,6 +692,107 @@ let declarations = tools.json()?;
+        max_length = "(number) Maximum output length",
+    },
+}
+
+function Tool.call(args)
+    return args.text:sub(1, args.max_length) .. "..."
+end
+```
+
+### Workspace structure
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)